app/views/taxon_changes/edit.html.haml

```diff
@@ -3,7 +3,12 @@
     =t :editing
     =t @taxon_change.class.name.underscore
     = surround '(', ')' do
-      = @taxon_change.committed? ? t(:committed_on, date: l( @taxon_change.committed_on ) ) : t(:draft)
+      - if @taxon_change.status_committed?
+        = t(:committed_on, date: l(@taxon_change.committed_on))
+      - elsif @taxon_change.status_withdrawn?
+        = t(:withdrawn)
```

Comment:
There is no root-level I18n key of `:withdrawn` - there's one at `views.taxon_changes.withdrawn`